src/Tabs/Tabs.test.jsx

```diff
@@ -3,16 +3,16 @@ import { shallow } from 'enzyme';
 import Tabs from './index';
 
 const props = {
-  labels: [
-    'first',
-    'second',
-    'third',
-  ],
-  children: [
-    <div>first</div>,
-    <div>second</div>,
-    <div>third</div>,
-  ],
+  tabs: [{
+    label: 'First',
+    content: 'I am the first',
```

Comment:
Looks like you're using the key name of `content` here but `panel` elsewhere. I think `content` makes a little more sense than `panel` as it describes what it does a little more clearly. We should make sure we're using `content` everywhere `panel` is used.

Reply:
Sounds good! Changed panels to contents except for the classNames